index.html

```diff
@@ -225,6 +225,38 @@ <h3 class="nav-sub-sub-heading weight--500 text--info mt-12">
                     <div class="hr"></div>
                 </article>
             </section>
+
+            <section class="contact-bg-color">
+                <article class="container ">
+                    <aside>
+                        <div class="weight--700 contact-title">
+                            Contact me
+                        </div>
+                        <p class="mt-12 weight--400 contact-sub-title">
+                            If you have an application you are interested in developing, a feature that you need built
+                            or a project that needs
+                            coding. I’d love to help with it
+                        </p>
+                    </aside>
+
+                    <aside>
+                        <form action="https://formspree.io/f/xknedbwy" method="post" class="contact-form">
+                            <div>
+                                <input type="text" maxlength="30" required name="fullName" placeholder="your full name"
+                                    class="weight--400">
+                            </div>
+                            <div>
+                                <input type="email" required name="email" placeholder="your name" class="weight--400">
```

Comment:
- [ ] I think this input is to receive the email of a user. Kindly use the correct placeholder text to give users an emphasis on what to put in that input. 😎️